djpaddle/models.py

```diff
@@ -57,45 +47,52 @@ class Meta:
 
     @classmethod
     def api_list(cls):
-        return api.retrieve(uri=cls.PADDLE_URI_LIST)
+        return PADDLE_CLIENT.list_plans()
 
     @classmethod
     def sync_from_paddle_data(cls, data):
-        pk = data.pop("id", None)
-        initial_price = data.pop("initial_price", [])
-        recurring_price = data.pop("recurring_price", [])
+        pk = data.pop("id")
+        initial_price = data.pop("initial_price", {})
+        recurring_price = data.pop("recurring_price", {})
 
-        plan, _ = cls.objects.get_or_create(pk=pk, defaults=data)
+        plan, __ = cls.objects.get_or_create(pk=pk, defaults=data)
```

Comment:
It's because the python interpreter stores the last expression value to the special variable called `_`. 
As `_` is technically already a variable it can be dangerous to use and can lead to some odd behaviour such as:
```
>>> 10
10
>>> _
10
>>> t, _ = [1, 2]
>>> _
2
>>> 10
10
>>> _
2
````

Using a double underscore on the other hand isn't a special variable so it's less likely it will be misused but is read the same.

You could take this to the extreme though and do something like:
(but that's so verbose it think it's a little confusing.)
```
plan, created = cls.objects.get_or_create(pk=pk, defaults=data)
del created
```

In reality it doesn't really matter though and the change was just force of habit. I'm happy to turn it back if you would like.